manifests/service.pp

```diff
@@ -120,8 +120,7 @@
   }
 
   $_manage_service = $manage_service ? {
-    false   => undef,
-    true    => 'Service[docker]',
+    true    => Service['docker'],
     default => undef,
```

Comment:
@jovandeginste  Thanks I have just forced a push with this change. Is it looking good ?

I am actually a bit surprised as the PR did not change anything in that regard. I mean it was `undef` before.